src/components/UserList/Row.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+} from 'react-native';
+
+class Row extends Component {
+  render() {
+    return (
+      <View style={styles.row}>
+        <Image style={styles.photo}
+          source={{ uri: 'http://lorempixel.com/64/64/cats/' }} />
+        <View style={styles.imageSeperator}></View>
+        <View style={styles.userInformation}>
+          <Text style={styles.name}>Hyunchan Kim</Text>
```

Comment:
This seems like a mock data. Could you add a comment/TODO here to replace this with real data from backend?